templates/default/fulldoc/html/js/full_list.js

```diff
@@ -154,10 +171,12 @@ function partialSearch(searchString, offset) {
 function searchDone() {
   searchTimeout = null;
   highlight();
-  if ($('#full_list li:visible').size() === 0) {
+  var found = $('#full_list li:visible').size();
+  if (found === 0) {
     $('#noresults').text('No results were found.').hide().fadeIn();
   } else {
-    $('#noresults').text('').hide();
+    // This is read out to screen readers
+    $('#noresults').text('There are ' + found + ' results.').hide().fadeIn();
```

Comment:
I wanted to change noresults to searchresults or something, but I figure I cannot change this for existing templates out there. I've just updated it to output the results at the bottom of the list, and it's read out to screen readers.